azure-pipelines.yml

```
@@ -0,0 +1,69 @@
+trigger:
+  branches:
+    include:
+      - cccs-*
+      - feature/CLDN-*
+
+pool:
+  vmImage: 'ubuntu-latest'
```

I'm a big fan of using quotes only when they're required in YAML
```suggestion
  vmImage: ubuntu-latest
```